repository/file.go

```diff
@@ -9,7 +9,7 @@ import (
 // File repository.
 type File interface {
 	Find(ctx context.Context, id int64) (*model.File, error)
-	FindAll(ctx context.Context) ([]model.File, error)
+	FindAll(ctx context.Context, page, limit uint64) ([]model.File, uint64, error)
```

Comment:
Daca vrei si capabilitati de filtrare poti sa folosesti definitia de mai jos, si orice prop al lui File care nu e default va fi folosit pt filtrare
```
	FindAll(ctx context.Context, query models.File, page, limit uint64) ([]model.File, uint64, error)
```

Reply:
te referi ca in .Where() sa folosesc asta?

Reply:
ma refer ca de obicei , pe o list table poti filtra dupa 1 sau mai multe coloane, daca vrei functionalitatea asta te poti folosi de model pt ca modelul are toate campurile dupa care poti face o filtrare.